joblib/_memmapping_reducer.py

```diff
@@ -361,7 +361,7 @@ def __reduce__(self):
         # needs to be pickled but the _WeakArrayKeyMap need to be skipped as
         # it's only guaranteed to be consistent with the parent process memory
         # garbage collection.
-        args = (self._max_nbytes, self._temp_folder_resolver, self._mmap_mode,
+        args = (self._max_nbytes, None, self._mmap_mode,
```

Comment:
Could you please also update the above comment to explain that we can also skip `self._temp_folder_resolver` as it's only needed when pickling arrays to be send to children processes.

Reply:
No idea either.